openshift-ci/Dockerfile.tools

```diff
@@ -28,6 +28,7 @@ RUN yum install epel-release -y \
     yamllint \
     python36-virtualenv \
     jq \
+    gcc \
```

Comment:
This one should go to the separate PR :(
Openshift-ci pulls Dockerfile.tools from master. So we need to merge it to master first - https://github.com/codeready-toolchain/toolchain-e2e/pull/35
Then merge it to this PR.

But tbh I don't quite understand why gcc is required now... Do you have any idea?

Reply:
ah ok, I'll open a separate PR for this change, then (anyways, it's out of the current PR's scope)
I wanted to add this change because of the e2e test error I got:
```
go: finding sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
go: finding k8s.io/gengo v0.0.0-20190907103519-ebc107f98eab
# runtime/cgo
exec: "gcc": executable file not found in $PATH
FAIL	github.com/codeready-toolchain/toolchain-e2e/test/e2e [build failed]
FAIL
```
but I'm not sure why we're seeing this now :/

Reply:
opened https://github.com/codeready-toolchain/toolchain-e2e/pull/36

Reply:
ah, but now I see you already opened https://github.com/codeready-toolchain/toolchain-e2e/pull/35! I'll close https://github.com/codeready-toolchain/toolchain-e2e/pull/36, then

Reply:
Yeah.. I saw the error. It's not introduced by this PR. It actually started to fail after merging https://github.com/codeready-toolchain/toolchain-e2e/pull/33 but not sure why. Maybe operator-sdk 0.11.0 requires it...

Reply:
Yeah, and I thought it was because of Kube 1.14 but https://github.com/codeready-toolchain/toolchain-e2e/pull/32 is not merged yet, so that might be operator-sdk 0.11, indeed.